projects/dendrites/permutedMNIST/experiments/centroid.py

```diff
@@ -54,10 +54,14 @@ class CentroidFigure1BExperiment(CentroidFigure1B,
     pass
 
 
+class CentroidExperimentPerTask(EvalPerTask, CentroidExperiment):
+    pass
```

Comment:
`CentroidExperimentPerTask` sounds misleading, I would recommend calling it `CentroidExperimentEvalPerTask`, but not a big deal.